raft/log.go

```diff
@@ -436,6 +523,12 @@ func (l *Log) writeTerm(term uint64) error {
 	return ioutil.WriteFile(l.termPath(), b, 0666)
 }
 
+// setTerm sets the current term if it's higher.
```

Comment:
This comment does not look correct. I don't see any check within the code. It just sets the term unconditionally.